spec/requests/api/v1/course_spec.rb

```diff
@@ -49,12 +49,13 @@ def get_course_codes_from_body(body)
           provider: provider,
           age: 2.hours.ago)
 
-        course.site_statuses.first.update(
+        course.site_statuses.first.update_attributes(
           vac_status: :full_time_vacancies,
           publish: 'Y',
           status: :running,
           applications_accepted_from: "2018-10-09 00:00:00"
         )
+        course.update_attributes changed_at: 2.hours.ago
```

Comment:
I think it might be good to use `update_columns` here, which bypasses validations and hooks. Although, and this might echo your own comments to me about how do our factories, could we also create the `SiteStatus` as part of the create of the `Course`, maybe by passing it in instead of the `site`? That would avoid having to reset `changed_at` I think.

Reply:
I think `#update` is fine for that, it doesn't fire any of those. As for changing the creation, it still causes the callback to fire, so the `changed_at` will be the current time.

I feel like the root cause is doing this via callback instead of explicitly when we want to publish the course to UCAS.

Reply:
I with `govuk-lint` forced us to use parenthesis here i.e. `course.update(changed_at: 2.hours.ago)`

Reply:
This should changed to `#update`, then. I wonder if the approach I outlined in #209 would benefit here.

Reply:
Oops, my bad, done now.